src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/EntityAIMeleeGuard.java

```diff
@@ -58,11 +58,6 @@
     private static final double BASIC_VOLUME = 1.0D;
 
     /**
-     * Experience the guard receives each shot arrow.
-     */
-    private static final double XP_EACH_HIT = 0.2;
-
-    /**
      * Base speed of the guard he follows his target.
      */
     private static final int BASE_FOLLOW_SPEED = 1;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "XP_EACH_HIT" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)